### PR TITLE
tor: update to 0.4.7.10

### PR DESCRIPTION
**Pull request by PolynomialDivision**
Release Notes:
https://forum.torproject.net/t/urgent-stable-release-0-4-5-14-0-4-6-12-and-0-4-7-10

Maintainer: 
Compile tested: Bananapi BPI-R64
Run tested: Bananapi BPI-R64

### CODE REVIEW

File: net/tor/Makefile [hunk 1]
```diff
@@ -8,13 +8,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=tor
-PKG_VERSION:=0.4.7.8
+PKG_VERSION:=0.4.7.9
```
Comment (BKPepe) [inline net/tor/Makefile:11]: I wonder, why not update it rather to 0.4.7.10?

Comment (PolynomialDivision) [reply net/tor/Makefile]: Thanks. :) Updated.

**Review by BKPepe — APPROVED** [review]
Looks good to me from my side.

**Comment by PolynomialDivision** [comment]
Due to the urgency of the update I think it is fine to merge it.

**Pull request merged**